vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -75,6 +75,20 @@ GOPATH: "{{ lookup('env', 'GOPATH')}}"
 IRONIC_TLS_SETUP: "{{ lookup('env', 'IRONIC_TLS_SETUP') | default('true', true)}}"
 IRONIC_BASIC_AUTH: "{{ lookup('env', 'IRONIC_BASIC_AUTH') | default('true', true)}}"
 
+# Environment variables for upgrade workflow
+IRONIC_IMAGE_TAG: "master"
+CAPIRELEASE_HARDCODED: "v0.3.8"
+UPGRADED_K8S_VERSION_1: "v1.21.0" # to be changed
+UPGRADED_K8S_VERSION_2: "v1.21.0"
+NUM_IRONIC_IMAGES: 8
+USER: "ubuntu"
+M3PATH: "{{ lookup('env', 'M3PATH') }}"
+CAPM3PATH: "{{M3PATH}}/cluster-api-provider-metal3"
+CAPM3_REL_TO_VERSION: "v0.4.1"
+CAPI_REL_TO_VERSION: "v0.3.14"
+CAPM3_ALPHA_VERSION: "v1alpha4"
```

Comment:
is there any problem with re-using existing vars? (check lines 57-58)

Reply:
These vars are the first ones (metal3-dev-env is creating with these). Then we upgrade to the recent versions.

Reply:
Not sure if I got the point here and also we are not upgrading to recent versions of CAPI&CAPM3 in the tests, do we? I was referring to `CAPM3_ALPHA_VERSION` and `CAPI_ALPHA_VERSION` and if we need them redefined where we already have them defined [here](https://github.com/metal3-io/metal3-dev-env/pull/634/files#diff-a47712e396d2d1f37a04cb69d32bfdc0db4522e4dbab9faf46fff5a1d0b89ab8R57-R58)?